src/Service/CarService.php

```diff
@@ -21,12 +21,13 @@ public function __construct(
         private readonly CarRepository $carRepository,
         private readonly EntityManagerInterface $entityManager,
         private readonly HttpClientInterface $httpClient,
-        private readonly Security $security, 
-        private readonly LoggerInterface $logger,
         private readonly ValidatorInterface $validator,
-        string $apiHost
-    ) {
-        $this->apiHost = $apiHost;
+        private readonly LoggerInterface $logger,
+        private readonly Security $security, 
+        string $apiHost,
+ )  {
+ $this->apiHost = $apiHost;
```

Comment:
since you're refactoring, declare $apiHost directly as private readonly in constructor method header